tests/Integration/Momento.Sdk.Tests/Topics/TopicTest.cs

```diff
@@ -143,10 +143,67 @@ public async Task PublishAndSubscribe_String_Succeeds()
         Assert.Equal(valuesToSend.Count, consumedMessages.Count);
         for (var i = 0; i < valuesToSend.Count; ++i)
         {
-            Assert.Equal(((TopicMessage.Text)consumedMessages[i]).Value, valuesToSend[i]);
+            var textMessage = (TopicMessage.Text)consumedMessages[i];
+            Assert.Equal(textMessage.Value, valuesToSend[i]);
+            Assert.Equal(textMessage.TopicSequenceNumber, i + 1);
         }
     }
 
+    [Fact(Timeout = 15000)]
+    public async Task PublishAndSubscribe_AllEventsString_Succeeds()
+    {
+        var topicName = Utils.NewGuidString();
+        var valuesToSend = new List<string>
+        {
+            "one",
+            "two",
+            "three",
+            "four",
+            "five"
+        };
+
+        var produceCancellation = new CancellationTokenSource();
+        //produceCancellation.CancelAfter(2000);
```

Comment:
nit: looks like this can be removed.

Reply:
Resolved in [dd6ab9d](https://github.com/momentohq/client-sdk-dotnet/pull/575/commits/dd6ab9dfc7a23d868ec5a4c4aad8939bf6cfa562)